backend/app/controllers/spree/admin/users_controller.rb

```diff
@@ -53,16 +53,6 @@ def update
         end
       end
 
-      def addresses
```

Comment:
I'm afraid we need to deprecate the endpoint before removing it straight away, as some users might have overridden it. You take a look at how we did something similar here: https://github.com/solidusio/solidus/pull/4388/files

Reply:
Yes, people may still have this path in their customized views. We should deprecate this instead (and not remove the route),

Reply:
@waiting-for-dev @kennyadsl  I added deprecation warning only to put endpoint `addresses`.

New get endpoint path `addresses` is the same as the old one